### PR TITLE
Inquire and add organization name to created CA.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context

This PR allows users to specify an organization name when creating a new CA in preflight. This makes the created CA distinguishable between different organizations/setups/departments.

/nocl

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-server/src/main/java/org/graylog/security/certutil/CaServiceImpl.java [hunk 1]
```diff
@@ -100,9 +100,9 @@ public CA get() throws KeyStoreStorageException {
     }
 
     @Override
-    public void create(final Integer daysValid, char[] password) throws CACreationException, KeyStoreStorageException, KeyStoreException {
-        final Duration certificateValidity = Duration.ofDays(daysValid == null || daysValid == 0 ? DEFAULT_VALIDITY: daysValid);
-        KeyStore keyStore = caCreator.createCA(passwordSecret.toCharArray(), certificateValidity);
+    public void create(String organization, final Integer daysValid, char[] password) throws CACreationException, KeyStoreStorageException, KeyStoreException {
```
Comment (luk-kaminski) [inline graylog2-server/src/main/java/org/graylog/security/certutil/CaServiceImpl.java:103]: All arguments could be final, not only `daysValid`.

File: graylog2-server/src/main/java/org/graylog/security/certutil/CertutilCa.java [hunk 2]
```diff
@@ -62,7 +62,7 @@ public void run() {
             console.printLine("Generating datanode CA");
 
             final Duration certificateValidity = Duration.ofDays(10 * 365);
-            KeyStore caKeystore = caCreator.createCA(password, certificateValidity);
+            KeyStore caKeystore = caCreator.createCA("Graylog CA", password, certificateValidity);
```
Comment (luk-kaminski) [inline graylog2-server/src/main/java/org/graylog/security/certutil/CertutilCa.java:65]: "Graylog CA" magic string could be promoted and become a constant.

**Review by luk-kaminski — APPROVED** [review]
Backend looks good, thx! 
(only really minor comments left)

**Review by linuspahl — APPROVED** [review]
Frontend looks good to me.

**Pull request merged**